app/controllers/katello/remote_execution_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Katello
+  class RemoteExecutionController < JobInvocationsController
```

Comment:
It mostly uses the present UI in JobInvocationsController (from remote execution). This provides the mapping between the Katello UI and the remote execution